addons/portal/static/src/views/list/portal_wizard_user_list_renderer.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { ListRenderer } from "@web/views/list/list_renderer";
+
+const { useSubEnv } = owl;
+
+export class PortalWizardUserListRenderer extends ListRenderer {
+    setup() {
+        super.setup();
+        this.isPortalActionOngoing = false;
+        const defaultOnClickViewButton = this.env.onClickViewButton;
+        useSubEnv({
+            onClickViewButton: async (params) => {
+                if (params.clickParams.name === 'action_refresh_modal' || this.isPortalActionOngoing) {
```

Comment:
Hello @Odoonan 👋 
I was taking a look at all overrides of `onClickViewButton` to make a better way of overriding it.
Doesn't this line make it never execute anything for `action_refresh_modal`? I am not sure what the expected behaviour is.
It feels like it's not on purpose :)
cc @awa-odoo

Reply:
Hi @Williambraecky, :hand: 
Yes, it is the purpose. This comes from the fact that we want the "email validity state" buttons to have the same display properties as buttons in list views (their tool tip message is functionally essential). In order to do so we set an action, strictly needed in the new framework. But since those button are mainly informative, we prevent the modal refreshing when pressing them, as it is not essential. It is not mandatory to prevent them though, but it is not obvious for the user these should reload the modal. I hope this is clearer?
Thanks for asking.

Reply:
Ok it makes more sense :)